<launch>
    <group ns="encoder">
        <param name="tickPerMeter" value="97.94" />
        <param name="minVelocity" value="0.05" />
    </group>
    <param name="batteryVoltage" value="7.85"/>
    <node pkg="my_robot" name="velocity_voltage_map" type="velocity_voltage_map" required="true">
        <param name="leftEncoder/pin" value="10"/>
        <param name="rightEncoder/pin" value="4"/>
        <param name="leftMotor/device" value="/dev/gpiopwm/18"/>
        <param name="rightMotor/device" value="/dev/gpiopwm/13"/>
    </node>
</launch>
